### DISCUSSION TITLE
Saving system

### DISCUSSION THREAD

**Discussion by Vsauce07** [Vdauphin/HeartsAndMinds#1320] [Question and answer] [answered]
**Hello,**
is there a way to change the saving system to the Vanilla one? I can't create a dedicated server since I only have one computer. Thank you in advance!

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> is there a way to change the saving system to the Vanilla one?

No

> I can't create a dedicated server since I only have one computer. 

You already have a dedicated server on your computer. Next to the arma3.exe you have a arma3server.exe 
Just setup it as you will do it on a dedicated server.
I am developing the H&M like that. It would be much more painfull to do it with a second computer.

Cheers

**Reply by Vsauce07** [reply]
Sorry for the dumb question, but can I use TADST to setup the server on the same PC?

**Reply by Vdauphin** [reply]
Never used this tool but if you look the video you see a arma.exe https://youtu.be/0GMbi0r0x8o?t=20
So look like he is running it from a client

**Reply by Vsauce07** [reply]
I created the server but the Headless slot it's empty. Is it normal?

**Reply by Vdauphin** [reply]
> Is it normal?

If you don't launch an headless, yes

**Reply by Vsauce07** [reply]
Thank you, everything worked. Btw, I have a question not related to the topic: is it possible to make a POW camp where if you put prisoners, it increases the reputation? It's a bit odd that after you arrest someone, you have to kill him or leave it there

**Reply by Vdauphin** [reply]
Start a new discussion :p

**Reply by Vsauce07** [reply]
Alright!
